tests/driver.c

```diff
@@ -39,11 +39,12 @@ void test_time_end(struct timespec* start, struct timespec* end, double* sum) {
 void test_generate_address(void) {
   char* json_result;
   double sum = 0;
+  status_t ret = SC_OK;
 
   for (size_t count = 0; count < TEST_COUNT; count++) {
     test_time_start(&start_time);
-    TEST_ASSERT_FALSE(
-        api_generate_address(&ta_core.tangle, &ta_core.service, &json_result));
+    ret = api_generate_address(&ta_core.tangle, &ta_core.service, &json_result);
+    TEST_ASSERT_EQUAL_INT32(ret, SC_OK);
```

Comment:
These changes should be in another commit. The only commit message doesn't even mention these.